HomeTask3.playground/Contents.swift

```diff
@@ -0,0 +1,67 @@
+import Darwin
+//Создать массив из кортежей (Int, String).
+
+let tuplesArray = [(1, "My name ist Valdemar"), (3, "Three"), (12, "Elizabeth"), (88, "Hello")]
+var tuplesArrayWithSquaredNumbers : [(Int, String)] = []
+
+//1. Пройти по элементам с помощью цикла for-in, создав Range от 0 до количества элементов массива, и возвести все Int в квадрат. (Лучше создать новый массив и результаты записать туда).
+
+for range in 0..<tuplesArray.count {
```

Comment:
I believe we can rename `range` to `index` because it's better describes what the variable does

Reply:
renamed